src/components/blipSelect/index.js

```diff
@@ -181,23 +185,36 @@ export class BlipSelect extends Component {
     }
     this.renderedOptionsList = this.optionsList.render(this._optionsListConfig())
     const isReadOnly = () => this.configOptions.mode === 'select'
-    const hasBulletClass = () =>
-      this.configOptions.mode === 'select' || this.configOptions.showBullet ? bpInputWithBulletClass : ''
+    const hideLabelClass = () =>
+      this.props.label ? '' : bpSelectHideLabelClass
+    const showArrowClass = () =>
+      this.configOptions.mode === 'select' ||
+      this.configOptions.showBullet ||
+      (this.optionsList.props.options && this.optionsList.props.options.length > 0)
+        ? bpSelectShowArrowClass
+        : ''
     return html`
-      <div class="bp-input-wrapper blip-select ${this.props.disabled ? 'bp-select-wrapper--disabled' : ''} ${hasBulletClass()}">
-        <label class="bp-label bp-c-rooftop">${this.props.label}</label>
-        <input placeholder="${this.configOptions.placeholder}"
-          class="blip-select__input bp-c-rooftop"
-          value="${this.props.inputValue}"
-          onfocus="${this._handleSelectFocus}"
-          onblur="${this._handleSelectBlur}"
-          onkeydown="${this._handleInputKeydown}"
-          onkeyup="${this._handleInputKeyup}"
-          oninput="${this._handleInputChange}"
-          onclick="${this._handleInputClick}"
-          data-target="${this.customSelectId}"
-          disabled="${this.isDisabled}"
-          readonly="${isReadOnly()}">
+      <div class="bp-input-wrapper blip-select ${this.props.disabled ? 'bp-select-wrapper--disabled' : ''}">
+        <div class="bp-select-shell">
+          <div class="bp-select-content">
```

Comment:
`bp-` preffix is related to shared blip-toolkit classes (`bp-input, bp-c-*, bp-bg-*...`). In this case, would be better to use BEM style for naming classes to keep the same idea of the rest of this component. Ex.: `blip-select__shell` and `blip-select__content`